lms/djangoapps/ccx/plugins.py

```diff
@@ -37,8 +37,6 @@ def is_enabled(cls, course, user=None):
         # Disable ccx coach tab for master courses if user is not allowed to create ccx.
         if (
             not is_ccx_course(course.id) and
-            # site configuration PCO_ENABLE_LICENSE_ENFORCEMENT is to control platform default.
-            configuration_helpers.get_value('PCO_ENABLE_LICENSE_ENFORCEMENT', False) and
             not run_extension_point(
                 'PCO_IS_USER_ALLOWED_TO_CREATE_CCX',
```

Comment:
**PART 3:**
`is_user_allowed_to_create_ccx` validates if Course Licensing is enable. This is the other usage I have seen. There are functions that check that Course Licensing and those that don't. The way I see things those validations should be made before calling any run_extension from the plugin. With that being said I ask you to use a variable `is_course_licensing_enable = run_extension_point('PCO_ENABLE_COURSE_LICENSING')` and use it correspondingly within the wrapper of this decorator `coach_dashboard` and there won't be need to check `is_course_licensing_enabled` in `is_user_allowed_to_create_ccx`. @anfbermudezme